app/web/src/store/components.store.ts

```diff
@@ -326,30 +326,12 @@ export function getPossibleAndExistingPeerSockets(
       (c) =>
         c.def.sockets
           ?.filter((peerSocket) => {
-            // Management inputs can only be connected to management outputs
```

Comment:
This logic determines possible connections on the Connections Panel. Before this change, it denied connections between management and non-management sockets, and between management sockets with non-matching schema. Now it denies connections between management and non-management sockets.